plume-models/src/safe_string.rs

```diff
@@ -19,13 +19,23 @@ lazy_static! {
     static ref CLEAN: Builder<'static> = {
         let mut b = Builder::new();
         b.add_generic_attributes(iter::once("id"))
-            .add_tags(iter::once("iframe"))
+            .add_tags(&[ "iframe", "video", "audio" ])
             .id_prefix(Some("postcontent-"))
             .url_relative(UrlRelative::Custom(Box::new(url_add_prefix)))
             .add_tag_attributes(
                 "iframe",
-                ["width", "height", "src", "frameborder"].iter().map(|&v| v),
-            );
+                [ "width", "height", "src", "frameborder" ].iter().map(|&v| v),
+            )
+            .add_tag_attributes(
+                "video",
+                [ "src", "title", "controls" ].iter(),
+            )
+            .add_tag_attributes(
+                "audio",
+                [ "src", "title", "controls" ].iter(),
+            )
+            .add_tag_attributes("div", [ "class" ].iter())
+            .add_tag_attributes("a", [ "class" ].iter());
```

Comment:
I'm not a big fan off allowing custom class, peoples could inject div/a with the right classes to mess up the interface. I'd prefer maybe an "unsafe" (not necessary in the sens of rust's unsafe) builder for SafeString, with the contract that things put in it must already be trusted